client/client_test.go

```diff
@@ -2,18 +2,22 @@ package client
 
 import (
 	"bytes"
+	"fmt"
 	"io/ioutil"
 	"net/http"
+	"path/filepath"
+	"reflect"
+	"runtime"
 	"strings"
 	"testing"
 
 	"github.com/3scale/3scale-porta-go-client/fake"
 )
 
 func TestNewAdminPortal(t *testing.T) {
-	_, err := NewAdminPortal("https", "www.test.com", 443)
```

Comment:
is `NewAdminPortal` being unittested somewhere else?

Reply:
pushed